datascience-notebook/.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Docker Test & Build
+
+"on":
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - name: Run yaml Lint
+        uses: actionshub/yamllint@master
```

Comment:
I never used GitHub Actions but I've heard that not pinning the version to the SHA may lead to potential security problems, e.g. the action code may change.

e.g.: https://julienrenaux.fr/2019/12/20/github-actions-security-risk/

I guess it's about the tradeoff between using the latest/greatest version of an action and being sure that the action never change.

I just thought it could be useful to at the very least mention in ☺️

Reply:
thankfully the person changing this is either a close friend of mine or me 😄 

That said we're always looking for more maintainers 😉